src/spi.rs

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Note that you'll want to make this the MPL header before merging

Reply:
Done in the Great Rebasing!